src/main/java/frc/robot/AutoReverseShootGroup.java

```diff
@@ -15,7 +15,7 @@
     public AutoReverseShootGroup(Robot robot) {
         super(
             new AutoDrive(robot, -.5, 0, 2), // drive backwards
-            new AutoShoot(robot, 3, .4) //shoot
+            new AutoShoot(robot, .4, 3) //shoot
```

Comment:
To clarify, this command group is only for running during the autonomous period right, not to be confused with the actual auto commands used to automate the shooting from the Hub and Wall

Reply:
Yes, this one runs in autonomous only. I'll rename it to clarify.